vivaldi/hooks/treeview-spacing.js

```diff
@@ -0,0 +1,184 @@
+// Change spacing of tree views (may need additional CSS)
+
+vivaldi.jdhooks.addStyle(`
+    :root {
+        --rowHeight: 24px;
+    }
+    .vivaldi-tree .tree-item .tree-row label {
+        line-height: var(--rowHeight);
+        min-height: var(--rowHeight);
+    }
+    .vivaldi-settings .settings-sidebar .button-category {
+        height: var(--rowHeight);
+    }
+    .panel#downloads {
+        --rowHeight: 54px;
+    }
+    .panel#downloads .DownloadItem {
+        --itemHeight: var(--rowHeight);
+    }
+`, 'treeview-spacing.js');
+
+// Odd name to avoid conflicts
+const jdHooksTvwSpcDetails = [
```

Comment:
You may use classical js trick to avoid name conflicts:

```javascript
(function(){
//the whole file content except the initial comment describing the mod
})()
```

Reply:
Thanks, will do. I don’t know why I didn’t use it.